plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -101,14 +110,27 @@ func (is *InternetSpeed) Gather(acc telegraf.Accumulator) error {
 		if err != nil {
 			return fmt.Errorf("upload test failed failed: %w", err)
 		}
+		_ = analyzer.Run(is.server.Host, func(packetLoss *transport.PLoss) {
```

Comment:
Same here, please either add a comment that the error is always `nil` _or_, even better, check the error as this is future proof...